wolfcrypt/src/port/nxp/se050_port.c

```diff
@@ -1210,12 +2815,9 @@ void se050_ed25519_free_key(ed25519_key* key)
         status = sss_key_object_get_handle(&newKey, key->keyId);
     }
     if (status == kStatus_SSS_Success) {
-        if ((key->flags & WC_ED25519_FLAG_DEC_SIGN) == 0) {
-            /* key was not used for signing, so release it */
-            sss_key_store_erase_key(&host_keystore, &newKey);
```

Comment:
This logic was here to allow a signing key to remain. Did the change to `kKeyObject_Mode_Persistent` resolve that?

Reply:
This behavior seemed odd to me, that we were taking control of erasing keys that the user may or may not want erased.

In the current code state, I opted to remove the auto-erase of any SE050 key by our free functions (ie: `se050_ed25519_free_key()`).  Users who then want the generated key freed from the key ID can call:

```
WOLFSSL_API int wc_se050_erase_object(word32 keyId); 
```

If you think this change might break some existing users, I can put this back for Ed25519.

Reply:
It was mainly to be able to run the wolfCrypt test. It won't break any existing users. Your change is fine.